app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -30,34 +30,27 @@ The main differences between courses are:
 
 * what you need to get onto the course
 * the qualifications awarded by the course
-* if the course is fee-paying or salaried 
+* if the course is fee-funded or salaried 
 * the type of course provider 
 * if the course is full-time or part-time 
 
 ## What you need to get onto the course
 
 Some training courses have specific eligibility requirements. 
 
-It's important to check the requirements for each course you apply to, so you do not waste time on an application you’re unlikely to be successful with. 
+It's important to check the requirements for each course you apply to on the training provider's course page. This is so you do not waste time on an application you’re unlikely to be successful with. 
 
 ### Your qualifications 
 
-All postgraduate teacher training courses require you to have: 
-
-* a degree in any subject 
-* GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary) 
-
-Most course providers will accept a degree grade of 2:2 or higher, but some may require you to have a 2:1. 
-
-Check each course provider’s requirements on their course page. You can also [find out about the qualifications you need to teach](/is-teaching-right-for-me/qualifications-you-need-to-teach).
+You need to have a degree in any subject to be able to teach. But the degree grade that providers will accept may vary, for example it may be a 2:1 or a 2:2. So you’ll need to check each course provider’s requirements on their course page. [Find out about the qualifications you need to teach](/is-teaching-right-for-me/qualifications-you-need-to-teach).
```

Comment:
```suggestion
You need to have a degree in any subject to be able to train to teach. But the degree grade that course providers will accept may vary. For example, most providers will accept a degree grade of 2:2 or higher, but some may require you to have a 2:1. So you’ll need to check each provider’s requirements on their course page. [Find out about the qualifications you need to teach](/is-teaching-right-for-me/qualifications-you-need-to-teach).
```
You need a degree to **train**; you need QTS in most schools to actually teach if that makes sense?

If we are going to add in degree grades, suggest we use the original text which gives a bit more context?